src/templates/static/legal/payment_agents.jsx

```diff
@@ -2,185 +2,182 @@ import React from 'react';
-                    <li>{it.L('include a disclaimer and/or notice that any intellectual property rights including any Trademark or slogan whatsoever belong to the Company and any unauthorised use is strictly prohibited.')}</li>
+                    <li>{it.L('Receive the approval of the Company to include any information in relation to the Company')}</li>
+                    <li>{it.L('Include a disclaimer and/or notice that any intellectual property rights, including any trademark or slogan whatsoever, belong to the Company and any unauthorised use is strictly prohibited')}</li>
                 </ul>
             </li>
             <li>{it.L('It is agreed that the Company shall not be responsible or liable for any matter arising out of or in relation to the use and operation of any website owned or used by the Agent. It is understood that the Agent shall indemnify the Company for all and any losses that the Company may suffer arising out of, or in relation to the use or operations of any website used or owned by the Agent.')}</li>
-            <li>{it.L('It is further agreed and understood that the Agent shall not use any domain name that includes the name [_1] and any derivation or variation of this name able to give the impression that the Company and the Agent are the same Person.', it.website_name)}</li>
-            <li>{it.L('The Agent shall not:')}
+            <li>{it.L('It is further agreed and understood that the Agent shall not use any domain name that includes the name [_1] and any derivation or variation of this name that might give the impression that the Company and the Agent are the same person.', it.website_name)}</li>
+            <li>{it.L('The Agent shall not do the following:')}
                 <ul className='bullet'>
-                    <li>{it.L('represent itself as a representative of the Company or as an authorised Person by the Company in its advertising activities;')}</li>
-                    <li>{it.L('use the Company\'s name or Trademark in its advertising activities without a written authorisation to be given by the Company;')}</li>
-                    <li>{it.L('misrepresent its services provided to the clients;')}</li>
-                    <li>{it.L('engage in misleading or illusory or deceptive conduct; ')}</li>
-                    <li>{it.L('engage in misleading or illusory or deceptive advertising or promises;')}</li>
-                    <li>{it.L('prepare and publish any material or place any advertisements which refer to the Company and its relationship with the Company without an authorisation to be given by the Company; ')}</li>
-                    <li>{it.L('abusively or fraudulently whatsoever use the Application Programme Interface (API) of [_1]', it.website_name)}</li>
+                    <li>{it.L('Represent itself as a representative of the Company or as an authorised person by the Company in its advertising activities')}</li>
```

Comment:
```suggestion
                    <li>{it.L('Represent themselves as a representative of the Company or as an authorised person by the Company in its advertising activities')}</li>
```

Reply:
@AzitaRassi Shouldn't also be " ...by the Company in their advertising activities", since the advertising activities belong to the agent?